src/cljs/main/broadfcui/auth.cljs

```diff
@@ -159,6 +159,34 @@
                                        :redirectUri (.. js/window -location -origin)})
                                :on-done #(swap! state assoc :hidden? true)})))))})
 
+(defn force-signed-in [{:keys [on-sign-in on-sign-out on-error]}]
+  (fn [auth-token]
+    (utils/ajax {:url (str "https://www.googleapis.com/oauth2/v3/tokeninfo?access_token=" auth-token)
+                 :on-done
+                 (fn [{:keys [status-code success? get-parsed-response raw-response]}]
+                   (if success?
+                     (let [{:keys [email sub]} (get-parsed-response)
+                           auth2 (clj->js
+                                  {:currentUser
+                                   {:get
+                                    (constantly
+                                     (clj->js
+                                      {:getAuthResponse
+                                       (constantly (clj->js {:access_token auth-token}))
+                                       :getBasicProfile
+                                       (constantly (clj->js {:getEmail (constantly email)
+                                                             :getId (constantly sub)}))}))
+                                    :listen (fn [])}
```

Comment:
Maybe, `(constantly nil)`, although perhaps you are trying to communicate that the return value is not used.

Reply:
Not intentional. I wan't sure what was the best way to express a no-op function but if you're happy with `(constantly nil)` then so am I.

Reply:
Hmm... I feel like there should be a better way to show this intention. We don't have many (any?) other places where there's a no-op, because the best way to do it would be to design the library to gracefully handle not being given a function. So we don't have an established pattern.

To me, `(constantly nil)` suggests that the return value is being used, and it's always `nil`. `(fn [])` reads better as a no-op, but also kinda looks like a mistake. Maybe we should define `(def no-op (fn []))` in utils so this can be `:listen utils/no-op`.

Not a very strong opinion though.

Reply:
I like this suggestion. Perhaps `(defn no-op [])`.